app/javascript/src/Plans/components/DefaultPlanSelectCard.jsx

```diff
@@ -10,6 +10,7 @@ import {
 } from '@patternfly/react-core'
 import { DefaultPlanSelect } from 'Plans'
 import { Spinner } from 'Common'
+import * as alert from 'utilities/alert'
```

Comment:
Nice one! I like this, at least we can change it to `import * as alert from 'jquery'` later

Reply:
My idea is to eventually remove "flash" from the UI and start using Patternfly's `Alert`. This way we'll only have to change the implementation of `utilities/alert`.

Reply:
Yes that is the way to go, please remove assets pipeline from the code, it should be handled by webpack